packages/dcos-integration-test/extra/conftest.py

```diff
@@ -240,6 +240,7 @@ def __init__(self, dcos_uri, masters, public_masters, slaves, public_slaves,
 
         # URI must include scheme
         assert dcos_uri.startswith('http')
+        self.protocol = dcos_uri.split(':')[0]
```

Comment:
If we want to start parsing up the uri, we should do with: https://docs.python.org/3/library/urllib.parse.html